### PR TITLE
SIG Platform Charters, Docs, and Working Group Info

### PR DESCRIPTION
**Pull request by taneem-ibrahim**
## Description
Initial set of charter, working group docs for SIG Platform

## How Has This Been Tested?
Self tested

## Merge criteria:

- [X ] The commits are squashed in a cohesive manner and have meaningful messages.
- [N/A ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: sig-platform/charter.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+
+- Work covered under DevEx(UI/UX, Dashboard) and MLOps
+
+
+## Members:
+* Chairs: 
+  * Taneem Ibrahim (Red Hat)
+  * Landon LaSmith (Red Hat)
+  * Samu Veloso Lopez (Red Hat)
+  * Gabe Goodheart (IBM)
```
Comment (LaVLaS) [inline sig-platform/charter.md:33]: We should default to the person's name with no employer and a link to their GitHub profile

File: wg-serving-integration/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+## Goals
+
+- Provide a framework for external and internal model serving solutions to be included in the Open Data Hub ecosystem in a cohesive and unified manner.
+
+## Meetings
+* Working Group Meeting: TBD
+  * [Meeting notes and Agenda] TBD.
+
+
+## Organizers
```
Comment (LaVLaS) [inline wg-serving-integration/README.md:17]: The proposal for a Serving Integration should be a collaboration between the Platform and ML Ops SIG. Does the ML Ops SIG have a representative?

File: sig-platform/charter.md (hunk 1 above)
Comment (taneem-ibrahim) [reply sig-platform/charter.md]: Updated. The k8s charters typically have company associations, so I followed that.

File: sig-platform/README.md [hunk 3]
```diff
@@ -1 +1,30 @@
...
+### Members
+The members of the SIG establish new subprojects, decommission existing subprojects, and resolve cross-subproject technical issues and decisions.
+
+- Chris Chase
+- JooHo Lee 
+- Sean Pryor 
+- Ben Herta 
+
+# Contact
+- [Slack](https://join.slack.com/t/odh-io/shared_invite/zt-18ptx7far-SWO4jkDbuA7Sq8Mut3JbcA)
```
Comment (LaVLaS) [inline sig-platform/README.md:29]: Can you  add the `#sig-platform` channel name to this? I think adding `...on #sig-platform` should work

File: sig-platform/OWNERS [hunk 4]
```diff
@@ -1,7 +1,12 @@
...
 approvers:
-  - add-names-here
+  - LaVLaS
+  - samuelvl
+  - gabe-l-hart
+  - taneem-ibrahim
 labels:
-  - sig/platform
-
+  - sig/platform
```
Comment (anishasthana) [inline sig-platform/OWNERS:12]: Missing newline? :-)

File: sig-platform/README.md [hunk 5]
```diff
@@ -1 +1,30 @@
...
+
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
+- Taneem Ibrahim
+- Landon LaSmith
+- Samu Veloso Lopez
+- Gabe Goodheart
+
+### Members
+The members of the SIG establish new subprojects, decommission existing subprojects, and resolve cross-subproject technical issues and decisions.
```
Comment (anishasthana) [inline sig-platform/README.md:21]: Should Vaishnavi be on this list?

File: sig-platform/README.md [hunk 6]
```diff
@@ -1 +1,30 @@
-To be created by SIG...
\ No newline at end of file
+# ODH Platform Special Interest Group
+
+SIG Platform covers odh operator, core components, ecosystem integration, authetication, and monitoring
```
Comment (anishasthana) [inline sig-platform/README.md:3]:
```suggestion
SIG Platform covers the ODH operator, odh-core components, ecosystem integration, authentication, and monitoring.
```

File: sig-platform/charter.md [hunk 7]
```diff
@@ -1 +1,43 @@
-To be created by SIG...
\ No newline at end of file
+# Charter
+
+This charter adheres to the conventions described in the [Open Data Hub Charter Guide](../sig-charter-guide.md) and uses the Roles and Organization Management outlined in [sig-goverance](../sig-governance.md).
+ 
+## Scope
+
+The Platform SIG maintains the core ODH operator, and establishes the standards and best practices for deployments, ecosystem integration, builds, monitoring, and secruity.
```
Comment (anishasthana) [inline sig-platform/charter.md:7]:
```suggestion
The Platform SIG maintains the core ODH operator, and establishes the standards and best practices for deployments, ecosystem integration, builds, monitoring, and security. 
```

File: sig-platform/charter.md [hunk 8]
```diff
@@ -1 +1,43 @@
...
\ No newline at end of file
+# Charter
+
+This charter adheres to the conventions described in the [Open Data Hub Charter Guide](../sig-charter-guide.md) and uses the Roles and Organization Management outlined in [sig-goverance](../sig-governance.md).
+ 
+## Scope
+
+The Platform SIG maintains the core ODH operator, and establishes the standards and best practices for deployments, ecosystem integration, builds, monitoring, and secruity. 
+
+body of expertise necessary to ensure architectural consistency over time.
```
Comment (anishasthana) [inline sig-platform/charter.md:9]: Is this supposed to be here?

File: sig-platform/charter.md [hunk 9]
```diff
@@ -1 +1,43 @@
...
+
+
+## Members:
+* Chairs: 
+  * Taneem Ibrahim
+  * Landon LaSmith
+  * Samu Veloso Lopez
+  * Gabe Goodheart
+  
+* Members:
```
Comment (anishasthana) [inline sig-platform/charter.md:35]: This list doesn't match the list from above.

File: wg-serving-integration/OWNERS [hunk 10]
```diff
@@ -0,0 +1,6 @@
+reviewers:
+  - wg-serving-integration-leads
+approvers:
+  - wg-serving-integration-leads
+labels:
+  - wg/serving-integration
```
Comment (anishasthana) [inline wg-serving-integration/OWNERS:6]: Missing newline

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: sig-platform/README.md (hunk 5 above)
Comment (LaVLaS) [reply sig-platform/README.md:21]: Yes

**Comment by LaVLaS** [comment]
LGTM once the comments from @anishasthana is resolved

File: sig-platform/OWNERS (hunk 4 above)
Comment (taneem-ibrahim) [reply sig-platform/OWNERS:12]: Hi @anishasthana Sorry I see it with a new line in my editor. What exactly is missing here?

File: sig-platform/charter.md (hunk 9 above)
Comment (taneem-ibrahim) [reply sig-platform/charter.md:35]: @anishasthana List of people?

File: sig-platform/OWNERS (hunk 4 above)
Comment (anishasthana) [reply sig-platform/OWNERS:12]: That's weird. Here's the diff I'm seeing in the GitHub UI 
image

File: sig-platform/charter.md (hunk 9 above)
Comment (anishasthana) [reply sig-platform/charter.md:35]: Yeah. Huh, I could have sworn it looked a little different earlier. It matches now.

File: sig-platform/charter.md [hunk 11]
```diff
@@ -1 +1,28 @@
...
+The Platform SIG maintains the core ODH operator, and establishes the standards and best practices for deployments, ecosystem integration, builds, monitoring, security and ensures architectural consistency over time.
+
+## In scope
+
+- Standards for telemetry, integrations, testing, builds, DevOps, and Security
+- Deployments (embedded ODH, edge deployments, airgaped, embedded) scalability
+- ODH Operator
+- ODH Core Component Integration
+- Code, Binaries, Design and Services Covered
+- [Opendatahub-io](https://github.com/opendatahub-io/) and [Opendatahub-io-contrib](github.com/opendatahub-io-contrib) GitHub orgs
```
Comment (LaVLaS) [inline sig-platform/charter.md:16]: Missing protocol so github markdown thinks it is a relative link

```suggestion
- [Opendatahub-io](https://github.com/opendatahub-io/) and [Opendatahub-io-contrib](https://github.com/opendatahub-io-contrib) GitHub orgs
```

**Review by anishasthana — APPROVED** [review]

**Pull request merged**